### PR TITLE
added questions in python-quiz.ts

### PR DESCRIPTION
**Pull request by kalashjain23**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

#15 
Added questions related to os module and cmd module in python.

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+  },
+  {
+    Question: 
+      "In Python, which module provides a way of using operating system dependent fuctionality?",
+    Answer: "os",
+    Distractor1: "sys",
+    Distractor2: "builtins",
+    Distractor3: "io",
+    Explanation: 
+      "The os module in Python provides functions for interacting with the operating system.",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:21]:
```suggestion
      "The Python os module is a set of functions used within the operating system. Some of these functions include fetching directory contents or creating new directories.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by kalashjain23** [comment]
Hey! I've made the required changes!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**